training/pytorch/structured/custom_containers/base/trainer/experiment.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the \"License\");
+# you may not use this file except in compliance with the License.\n",
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an \"AS IS\" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+import inputs
+import model
+
+
+def train(sequential_model, train_loader, criterion, optimizer, epoch):
+    """Create the training loop"""
```

Comment:
Add method documentation:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods